app/models/book.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class Book < ApplicationRecord
```

Comment:
Please add the validations specificated in the card here

Reply:
What validations you refer? In the card is only specified that all fields should be required. Setting them to ```NOT NULL``` doesnt cover that?

Reply:
Check http://guides.rubyonrails.org/active_record_validations.html . It's okey to set them as not null, but if someone sent, for example an empty string, the not null validation will accept the input. Apart from that, if we sent null, the api will respond with an exception and a 500 status code because of an unexpected sql value, the idea is to be able to handle those errors and you can do that with the RoR validators